### PR TITLE
Add french company siret and siren generator

### PR DESCRIPTION
**Pull request by hulous**
Hello 

I need to generate fake 'Siret' and 'Siren' in factories for french companies.
I see [Faker did it quite well... so I take that](https://github.com/faker-ruby/faker/blob/master/lib/faker/default/company.rb#L216-L244). :roll_eyes: 

I need to use luhn_check method from ssn_se. So I move it in the ModuleUtils.

Thank you. :wink:

### CODE REVIEW

File: lib/ffaker/company_fr.rb [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+    def suffix
+      fetch_sample(SUFFIXES)
+    end
+
+    ##
+    # @faker.version 1.8.5
+    # Get a random French SIREN number. See more here https://fr.wikipedia.org/wiki/Syst%C3%A8me_d%27identification_du_r%C3%A9pertoire_des_entreprises
+    def french_siren_number
+      base = (1..8).map { rand(10) }.join
```
Comment (marocchino) [inline lib/ffaker/company_fr.rb:26]:
```suggestion
      base = FFaker.numerify('########')
```

File: lib/ffaker/company_fr.rb [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+      base + luhn_check(base).to_s
+    end
+
+    ##
+    # @faker.version 1.8.5
+    # Produces a company french siret number.
+    def french_siret_number
+      location = rand(100).to_s.rjust(4, '0')
```
Comment (marocchino) [inline lib/ffaker/company_fr.rb:34]: https://github.com/rubocop/ruby-style-guide#random-numbers
i proper use rand(range) notation for clearfy.
```suggestion
      location = rand(0..99).to_s.rjust(4, '0')
```

File: lib/ffaker/utils/module_utils.rb [hunk 3]
```diff
@@ -35,5 +35,25 @@ def underscore(string)
     def unique(max_retries = 10_000)
       @unique_generator ||= FFaker::UniqueUtils.new(self, max_retries)
     end
+
+    # http://en.wikipedia.org/wiki/Luhn_algorithm
+    def luhn_check(number)
```
Comment (marocchino) [inline lib/ffaker/utils/module_utils.rb:40]: https://github.com/ffaker/ffaker/blob/e6cfa33d68339779d6e1906b024c19d77af209af/scripts/reference.rb#L28
We would appreciate it if you add it to this exclude list.  😄

**Comment by marocchino** [comment]
Thanks!

**Comment by hulous** [comment]
Thanks to you FFaker is awesome !

**Pull request merged**